assemble/conf/templates/accumulo-env.sh

```diff
@@ -33,24 +48,22 @@ JAVA_OPTS=("${ACCUMULO_JAVA_OPTS[@]}" '-XX:+UseConcMarkSweepGC' '-XX:CMSInitiati
 ## JVM options set for individual applications
 case "$ACCUMULO_CMD" in
 master)  JAVA_OPTS=("${JAVA_OPTS[@]}" ${masterHigh_masterLow}) ;;
+monitor) JAVA_OPTS=("${JAVA_OPTS[@]}" ${monitorHigh_monitorLow}) ;;
 gc)      JAVA_OPTS=("${JAVA_OPTS[@]}" ${gcHigh_gcLow}) ;;
 tserver) JAVA_OPTS=("${JAVA_OPTS[@]}" ${tServerHigh_tServerLow}) ;;
-monitor) JAVA_OPTS=("${JAVA_OPTS[@]}" ${monitorHigh_monitorLow}) ;;
 shell)   JAVA_OPTS=("${JAVA_OPTS[@]}" ${shellHigh_shellLow}) ;;
 *)       JAVA_OPTS=("${JAVA_OPTS[@]}" ${otherHigh_otherLow}) ;;
 esac
-export JAVA_OPTS
 
-## Accumulo logs directory. Referenced by logger config.
-export ACCUMULO_LOG_DIR="${ACCUMULO_LOG_DIR:-$ACCUMULO_HOME/logs}"
-## Hadoop installation
-export HADOOP_PREFIX="${HADOOP_PREFIX:-/path/to/hadoop}"
-## Hadoop configuration
-export HADOOP_CONF_DIR="${HADOOP_CONF_DIR:-${HADOOP_PREFIX}/etc/hadoop}"
-## Zookeeper installation
-export ZOOKEEPER_HOME="${ZOOKEEPER_HOME:-/path/to/zookeeper}"
-## See HADOOP-7154 and ACCUMULO-847
-export MALLOC_ARENA_MAX=${MALLOC_ARENA_MAX:-1}
+## JVM options set for logging
+JAVA_OPTS=("${JAVA_OPTS[@]}" "-Daccumulo.application=${ACCUMULO_CMD}" "-Daccumulo.log.dir=${ACCUMULO_LOG_DIR}" "-Daccumulo.local.hostname=$(hostname -s)")
```

Comment:
The 'accumulo.local.hostname' property is only used in the log4j properties files to set the hostname in file names for logs and to identify the host when logs are forwarded to the monitor.  I prefer the shorter hostname for these two cases but it's not a big issue for me and I will switch to `hostname -f` if you prefer.